### DISCUSSION TITLE
Copy/Paste to UI / table editor

### DISCUSSION THREAD

**Discussion by efpage** [directus/directus#10706] [Feature Requests]
### 1: copy / paste in table view

For convenience it would be nice to have a copy/paste option to the table view directly from excel. Referring to some sources this should not be too complicated to implement (see https://www.codeproject.com/Questions/1212303/How-copy-and-paste-excel-columns-to-HTML-table-wit).

### 2: Inplace editor

Simlar an in place editor for the table view would be very nice. I had lot´s of edits within directus, but changings to the item view is not the best option in any case.

**Comment by rijkvanzanten** [comment]
Ref https://github.com/directus/directus/discussions/3874
